cpp/contracts/feesmanager.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <eosio/eosio.hpp>
+#include <eosio/asset.hpp>
+
+using namespace eosio;
+
+CONTRACT feesmanager : public eosio::contract {
+public:
+    using contract::contract;
+
+    ACTION init( name security_council );
+
+    ACTION setallowance( name node, name token, uint64_t amount );
+
+    ACTION incallowance( name node, name token, uint64_t amount );
+
+    ACTION withdrawto( name node, name token );
+
+    ACTION withdrawto( name node, const std::vector<name>& tokens);
+
+private:
+    TABLE allowance {
+        name node;
+        name token;
+        uint64_t amount;
+
+        uint64_t primary_key() const { return node.value + token.value; }
+        
+        uint64_t by_token() const { return token.value; }
+    };
```

Comment:
Thinking about it this is not the "eos" way of doing this. If you check the `accounts` table in the `eosio.token.hpp` contract, you'll see this table:

```cpp
struct [[eosio::table]] account {
   asset    balance;

   uint64_t primary_key()const { return balance.symbol.code().raw(); }
};
```
Where you can access it like this:

```cpp
accounts from_acnts( get_self(), owner.value );
```

You can see the scope is restricted to `owner.value`, meaning there will be a different table for each account that will have some allowance. And it is indexed by the token symbol code, so each table per account will have a set of symbols with the specific balance.

Also, since EOS works with symbols rather then accounts for identifying tokens, I would use the `asset` type as done for the `balance` column in the `accounts` table.

**tl;dr** just copy the table above from `eosio.token.hpp` and rename `balance` to `allowance`